doc/user/inputs.rst

```diff
@@ -15,24 +14,39 @@ Settings file
   	activate) and all kind of modeling approximation settings (e.g. convergence criteria)
 
 Blueprints file
-	Contains dimensions and composition of the components/blocks/assemblies in your reactor systems, from fuel 
-  	pins to heat exchangers
+	Contains dimensions and composition of the components/blocks/assemblies in your reactor systems,
+    from fuel pins to heat exchangers
```

Comment:
I expect this might need to be indented.

Reply:
There were some indenting problems in this file (spaces mixed with tabs), I fixed them.